ui/app/components/job-status/update-params.js

```diff
@@ -44,7 +52,12 @@ export default class JobStatusUpdateParamsComponent extends Component {
       return this.rawDefinition.TaskGroups.map((tg) => {
         return {
           name: tg.Name,
-          update: tg.Update,
+          update: Object.keys(tg.Update).reduce((newUpdateObj, key) => {
+            newUpdateObj[key] = PARAMS_REQUIRING_CONVERSION.includes(key)
+              ? formatDuration(tg.Update[key])
+              : tg.Update[key];
+            return newUpdateObj;
+          }, {}),
         };
       });
     } else {
```

Comment:
Not necessarily part of the PR, but I think this function can be simplified a bit by returning early and removing the nested `return` in the arrow function:

```js
  get updateParamGroups() {
    if (!this.rawDefinition) {
      return null;
    }

    return this.rawDefinition.TaskGroups.map((tg) => ({
      name: tg.Name,
      update: Object.keys(tg.Update).reduce((newUpdateObj, key) => {
        newUpdateObj[key] = PARAMS_REQUIRING_CONVERSION.includes(key)
          ? formatDuration(tg.Update[key])
          : tg.Update[key];
        return newUpdateObj;
      }, {}),
    }));
  }
```

But definitely a style nit-pick 😄

Reply:
Returning early is a good suggestion / something I see often in our .go code. I'll try to implement it here and elsewhere as it comes up. Thank you!